website/integrations/services/owncloud/index.md

```diff
@@ -0,0 +1,210 @@
+            - **Client ID**: Use the value generated by authentik.
+            - **Client Secret**: Use the value generated by authentik.
+            - **Redirect URIs**:
+                - Strict: `https://owncloud.company/apps/openidconnect/redirect`
+
+            **Desktop Application**
+
+            - **Signing Key**: Select any available signing key.
+            - **Client ID**: Use the predefined value found in the [ownCloud admin manual](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-id).
+            - **Client Secret**: Use the predefined value found in the [ownCloud admin manual](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-secret).
+            - **Redirect URIs**:
+                - Regex: `http://localhost:\d+`
+                - Regex: `http://127.0.0.1:\d+`
+
+            **Android Application**
+
+            - **Signing Key**: Select any available signing key.
+            - **Client ID**: Use the predefined value found in the [ownCloud admin manual](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-id).
+            - **Client Secret**: Use the predefined value found in the [ownCloud admin manual](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-secret).
+            - **Redirect URI**:
+                - Strict: `oc://android.owncloud.com`
+
+            **iOS Application**
+
+            - **Signing Key**: Select any available signing key.
+            - **Client ID**: Use the predefined value found in the [ownCloud admin manual](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-id).
+            - **Client Secret**: Use the predefined value found in the [ownCloud admin manual](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-secret).
+            - **Redirect URI**:
+                - Strict: `oc://ios.owncloud.com`
+
+        - **Advanced protocol settings:**
+            - **Scopes**: Select the following scopes for each of the four application/provider pairs: `email`, `offline_access`, `openid`, `profile`.
+
+    - **Configure Bindings** (optional): To manage the listing and access to applications on a user's **My applications** page, you can optionally create a [binding](https://docs.goauthentik.io/docs/add-secure-apps/flows-stages/bindings/) between the application and a specific policy, group, or user. Keep in mind that if you do not specify any bindings, all users will have access to the application. For more information about user access, refer to our documentation about [authorization](https://docs.goauthentik.io/docs/add-secure-apps/applications/manage_apps#policy-driven-authorization) and [hiding an application](https://docs.goauthentik.io/docs/add-secure-apps/applications/manage_apps#hide-applications).
+
+### Service discovery
```

Comment:
Is there a more descriptive title? Also... can we specify that it is is the reverse proxy is configured within authentik? Should that be one of the regular steps in the "authentik configuration" section?

Reply:
One *can* configure a reverse proxy within authentik (@BeryJu ??), but if for sure this one is not configured there, then let's leave as is.